<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomeWork-1_1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>551</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Задание №1</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>11</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;a:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>12</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>b:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>11</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>c:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditA">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditB">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditC">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="resultLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="calculateButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Задание №2</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Side A:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Side B:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSideA">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditSideB">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAngle">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Units</string>
        </property>
        <widget class="QRadioButton" name="radioButtonRadians">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Radians</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonDegrees">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Degrees</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditResult">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>112</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Задание №3</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextSource">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>210</width>
          <height>130</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextDest">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>210</width>
          <height>130</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Push html</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>441</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
